contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -638,10 +638,11 @@ <h3 id="capability">Add a Capability to this Contribution</h3>
                     <div class="form-group">
                         <label class="required control-label col-sm-2">Data Deletion</label>
                         <div class="col-sm-10">
-                            <input class="form-inline" id="datadel_endpoint" name="dataDeletionEndpointDetails_deletionEndpoint"
+                            <input class="form-control" id="datadel_endpoint" name="dataDeletionEndpointDetails_deletionEndpoint"
                                    placeholder="Endpoint" type="text">
-                            <input class="form-inline" id="datadel_api" name="dataDeletionEndpointDetails_apiKey"
-                                   placeholder="Api Key" type="text">
+                            <input class="form-control" id="datadel_api" name="dataDeletionEndpointDetails_apiKey"
+                                   placeholder="Api Key" type="password">
+                            <i class="bi bi-eye-slash field_icon" id="togglePassword" style="cursor:pointer; position:absolute; margin-left:-20px; margin-top:-20px"></i>
```

Comment:
Same comment here. It might be better to move the inline styles to a new style class. Thanks.

Reply:
updated

Reply:
Thanks.